client/packages/host/webpack.config.js

```diff
@@ -8,6 +8,38 @@ const dependencies = require('./package.json').dependencies;
 const BundleAnalyzerPlugin =
   require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
+const fs = require('fs');
+
+const localPlugins = () => {
+  const plugins = [];
+  try {
+    fs.readdirSync('../plugins')
```

Comment:
I've added error handling to this, changed the shape and loaded the metadata from `plugin.json`